src/math-translator.js

```diff
@@ -499,14 +500,23 @@ function getEscapedDecimalSeparator(lang) {
  */
 function getOrderedPairRegexString(lang) {
     const katexColorMacros = `\\\\(?:${KATEX_BASE_COLORS.join('|')})[A-Z]?`;
-    // Assuming single-letter variables and numbers below 1000
-    // (without thousand separator)
+    // Assuming single-letter variables (or \pi) and numbers below 1000
+    // (without thousand separator) or combinations, such as '2\\pi' or '2.1b'
     const integer =
       `-?[0-9]+|-?${katexColorMacros}\\{-?[0-9]+\\}|-?${katexColorMacros}[0-9]`;
-    const variable = `[a-z]|${katexColorMacros}\\{[a-z]\\}`;
+    const variable = `\\\\pi|[a-z]|${katexColorMacros}\\{[a-z]\\}`;
     const decimal = getDecNumberRegexString(lang,
         /* don't include capture groups */ false);
-    const numberOrLetter = `${variable}|${decimal}|${integer}`;
+
+    const fracArgument = `(?:${variable}|${integer})`;
```

Comment:
Could it also contain `decimal`?

Reply:
It's a fraction so decimal in nominator or denominator doesn't make sense.